### ISSUE TITLE
[BUG]: resen crashes if you start a bucket without a valid image

### ISSUE THREAD

**Issue by ljlamarche** [EarthCubeInGeo/resen#10]
If you try to run start_bucket on a bucket without a valid image added, resen crashes.  It would be better if instead an error message displays that says the bucket cannot be started, i.e.

```
>>> Error: Cannot start bucket that does not have a valid image. 
                  To add image to bucket, use add_image.
```

**Comment by asreimer** [comment]
Confirmed:

    (env3.7) [asreimer@thomson ~]$ resen
        ___ ___ ___ ___ _  _ 
       | _ \ __/ __| __| \| |
       |   / _|\__ \ _|| .` |
       |_|_\___|___/___|_|\_|
    
    Resen 19.1.0 -- Reproducible Software Environment
    
    [resen] >>> create_bucket test
    [resen] >>> start_bucket test
    Traceback (most recent call last):
      File "/home/asreimer/virtualenvs/env3.7/bin/resen", line 10, in <module>
        sys.exit(main())
      File "/home/asreimer/virtualenvs/env3.7/lib/python3.7/site-packages/resen/resencmd.py", line 293, in main
        ResenCmd(res).cmdloop(intro)
      File "/usr/lib64/python3.7/cmd.py", line 138, in cmdloop
        stop = self.onecmd(line)
      File "/usr/lib64/python3.7/cmd.py", line 217, in onecmd
        return func(arg)
      File "/home/asreimer/virtualenvs/env3.7/lib/python3.7/site-packages/resen/resencmd.py", line 56, in do_start_bucket
        status = self.program.start_bucket(bucket_name)
      File "/home/asreimer/virtualenvs/env3.7/lib/python3.7/site-packages/resen/Resen.py", line 74, in start_bucket
        return self.bucket_manager.start_bucket(bucket_name)
      File "/home/asreimer/virtualenvs/env3.7/lib/python3.7/site-packages/resen/Resen.py", line 449, in start_bucket
        container_id = self.dockerhelper.create_container(**kwargs)
      File "/home/asreimer/virtualenvs/env3.7/lib/python3.7/site-packages/resen/Resen.py", line 645, in create_container
        repo, tag = image_name.split(':')
    AttributeError: 'NoneType' object has no attribute 'split'
    (env3.7) [asreimer@thomson ~]$ 
    
Need to add a check [here](https://github.com/EarthCubeInGeo/resen/blob/develop/resen/Resen.py#L441) for whether or not the bucket image name has been set yet or not.

**Comment by asreimer** [comment]
Fixed in #15
